client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -318,12 +319,13 @@ public <K, V, C> ShuffleHandle registerShuffle(int shuffleId, ShuffleDependency<
     if (dependency.partitioner().numPartitions() == 0) {
       LOG.info("RegisterShuffle with ShuffleId[" + shuffleId + "], partitionNum is 0, "
           + "return the empty RssShuffleHandle directly");
+      Broadcast<byte[]> ptsBd = RssSparkShuffleUtils.createPartShuffleServerMap(SparkContext.getOrCreate(),
```

Comment:
yes, I tried. But we cannot call 'SparkContext.getActive()' method from Java since it's package private in scala. And "registerShuffle" is called in Driver only after SparkContext was created. So we should be safe here and always get the active SparkContext.

Reply:
OK. You can extract a method to wrap `SparkContext.getOrCreate()` and add some comments to explain. Or we can use refection to call 'SparkContext.getActive()'.

Reply:
sounds good. let me add a method for comments.